Detectors/ITSMFT/common/reconstruction/src/Clusterer.cxx

```diff
@@ -34,6 +36,20 @@ Clusterer::Clusterer()
 //__________________________________________________
 void Clusterer::process(PixelReader &reader, TClonesArray &clusters)
 {
+  // This function will be removed as soon as the MFT code catches up. 
+  LOG(WARNING) << "DO NOT USE THIS DEPRECATED FUNCTION !" << FairLogger::endl;
```

Comment:
This will make the MFT clusterization stop working, just to ping to @bovulpes that MFT ClustererTask needs to be again copy/pasted from ITS with branches renamed (also the test macros).

Reply:
@shahor02 @bovulpes  In principal, this should not break the MFT clusterization. For the moment, the MFT ClustererTask simply calls this function, and keeps storing the MFT clusters in a TClonesArray.  

But !  There are no MC labels inside the clusters anymore.  So, yes, some cut&paste from ITS to MFT will certainly be needed.